classes/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly make sure the `rentals` has `attr_reader` and `attr_writer`, you can add `rentals` to `attr_accessor` to fix this 
```suggestion
  attr_accessor :title, :author, :rentals
```